includes/compat/class-log.php

```diff
@@ -251,11 +253,16 @@ public function file_download_log_get_post_meta( $value, $object_id, $meta_key,
 						$key = 'api_key';
 						break;
 					case 'user':
-						$key = 'user_id';
+						$key = 'customer_id';
+
 						break;
 				}
 
 				$value = $file_download_log->{$key};
+				if ( 'user' === $meta_key ) {
+					$customer = new EDD_Customer( $value );
```

Comment:
```suggestion
					$customer = new \EDD_Customer( $value );
```

Because this class is namespaced as `\EDD\Compat` and `EDD_Customer` is in the root namespace. So we need to prefix it with a `\` or import it. (Importing it would be adding `use EDD_Customer;` to the top of the file.